portfolio/src/main/webapp/style.css

```diff
@@ -103,11 +103,11 @@ a, a:hover {
         width: auto;  
     }
 }
-    @media screen and (min-width:483px) and (max-width: 1000px) {
-       .Navlinks{
+@media screen and (min-width:483px) and (max-width: 1000px) {
+    .Navlinks{
         margin-left: 9%;
     }
-       .items3{
-           display: none;
+    .items3{
+        display: none;
        }
     }
```

Comment:
This last line should be moved to the left, but that can be fixed in a future change.